x/leveragelp/keeper/position_close.go

```diff
@@ -91,6 +88,16 @@ func (k Keeper) ForceCloseLong(ctx sdk.Context, position types.Position, pool ty
 			return sdk.ZeroInt(), err
 		}
 	} else {
+		// Update position health
+		positionHealth, err := k.GetPositionHealth(ctx, position)
+		if err != nil {
+			return sdk.ZeroInt(), err
+		}
+		position.PositionHealth = positionHealth
+
+		// Update Liabilities
+		debt = k.stableKeeper.UpdateInterestStackedByAddress(ctx, position.GetPositionAddress())
+		position.Liabilities = debt.Borrowed
```

Comment:
Shouldn't we set position.LeveragedLpAmount by iterating over commitments?

Reply:
we are already updating it at `line 80`